### ISSUE TITLE
Broken indent expectations in template literals within classes

### ISSUE THREAD

**Issue by bdistin** [eslint/eslint#11017]
**Tell us about your environment**

* **ESLint Version:** 5.7.0
* **Node Version:** 10.12.0
* **npm Version:** 5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
	"extends": "eslint:recommended",
	"env": {
		"node": true,
		"es6": true
	},
	"parserOptions": {
		"ecmaVersion": 2018
	},
	"rules": {
		"indent": ["error", "tab", { "SwitchCase": 1 }]
	}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const reply = () => {
	// NoOp
};
const groups = [];

class Test {
	// Expected
	run() {
		return reply(`
			${groups.map(grp =>
				grp ? 'Enabled' : 'Disabled'
			)}
		`);
	}

	// Eslint Expects
	run2() {
		return reply(`
			${groups.map(grp =>
		grp ? 'Enabled' : 'Disabled'
	)}
		`);
	}
}

// Expected & Eslint Expects
reply(`
	${groups.map(grp =>
		grp ? 'Enabled' : 'Disabled'
	)}
`);

module.exports = Test;
```

```bash
eslint Test.js
```

**What did you expect to happen?**
*shown above in source code comments*
**What actually happened? Please include the actual, raw output from ESLint.**
*shown above in source code comments*
```bash
C:\workspace\Test.js
  11:1  error  Expected indentation of 2 tabs but found 4  indent
  12:1  error  Expected indentation of 1 tab but found 3   indent
```

**Comment by bdistin** [comment]
Bump. It's been a month with no labels, comments, references, a response of any kind to this issue.

**Comment by Aladdin-ADD** [comment]
oops, sorry for the delay!

it seems a bug to me, and I was able to repro it.  cc @not-an-aardvark

**Comment by platinumazure** [comment]
@not-an-aardvark Is there anything we could potentially do for this issue? If nothing else, can we give OP a workaround using `ignoredNodes`?

**Comment by not-an-aardvark** [comment]
I think this is the same issue as https://github.com/eslint/eslint/issues/10814 and https://github.com/eslint/eslint/issues/10932.

**Comment by bdistin** [comment]
I mean, there is indent legacy that still works properly... But as it's "legacy", that comes with the expectation that it will be removed at some point. Which is why I think it's still **very important** to fix this bug for when this indent rule becomes the only option.

Template literals aren't that uncommon...
